nixos/doc/manual/configuration/gpu-accel.xml

```diff
@@ -137,7 +137,7 @@ Platform Vendor      Advanced Micro Devices, Inc.</screen>
       <literal>:</literal>. For example:
 
       <screen><prompt>$</prompt> export \
-  VK_ICD_FILENAMES=`nix-build '&lt;nixpkgs&gt;' --no-out-link -A amdvlk`/share/vulkan/icd.d/amd_icd64.json</screen>
+  VK_ICD_FILENAMES=`nix-build '&lt;nixpkgs&gt;' --no-out-link -A amdvlk`/share/vulkan/icd.d/amd_icd64.json:`nix-build '&lt;nixpkgs&gt;' --no-out-link -A driversi686Linux.amdvlk`/share/vulkan/icd.d/amd_icd32.json</screen>
```

Comment:
I now looked at the rendered manual and this looks fairly complex as a first example as a first example of using `VK_ICD_FILENAMES`. I would just leave it as the original, most people should probably be using `extraPackages` anyway.